package.json

```
@@ -24,12 +24,12 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/utils": "^6.0.0",
+    "@typescript-eslint/utils": "^7.0.0",
     "tslib": "^2.3.1",
-    "tsutils": "^3.21.0"
+    "ts-api-utils": "^1.3.0"
   },
   "peerDependencies": {
-    "eslint": "^7.0.0 || ^8.0.0",
+    "eslint": "^8.0.0",
```

While we're in the area ... we _could_ do this, I think? So swapping (v7, v8) with (v8, v9)? 

```suggestion
    "eslint": "^8.0.0 || ^9.0.0",
```

---

Or, heh, I suppose that's already covered in #79.